mriqc/workflows/anatomical.py

```diff
@@ -81,15 +83,34 @@ def anat_qc_workflow(dataset, settings, mod='T1w', name='anatMRIQC'):
     to_ras = pe.Node(ConformImage(check_dtype=False), name='conform')
     # 2. Skull-stripping (afni)
     asw = skullstrip_wf(n4_nthreads=settings.get('ants_nthreads', 1), unifize=False)
+    if not use_fsl():
+        # remove fsl dependent nodes and reconnect a ants one
+        binarize = pe.Node(ThresholdImage(dimension=3, th_low=1.e-3, th_high=1e6, inside_value=1.0,
+                                          outside_value=0.0),
+                           name='binarize_ants')
+        asw.remove_nodes([asw.get_node('binarize')])
+        # copyheader is necessary because of precision error
+        # see https://github.com/ANTsX/ANTs/wiki/Inputs-do-not-occupy-the-same-physical-space, for more details
+        copyheader = pe.Node(CopyImageHeaderInformation(copy_origin=True, copy_direction=True, copy_spacing=True),
```

Comment:
Oh okay, it was used here. Please note that the name does not match (that's why this is failing).

Since you are using this to fix the output of ThresholdImage I can propose two solutions:
- We fix this directly in Nipype (more *administrative* burden, but I can probably have something working fast).
- We switch to nibabel for thresholding.

WDYT?